Bottom: 06454b3beb94c312085dbbb6273d70f049e9b3f3
Top:    d5432efc2e772754920ff314583f437b326bc298
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-18 16:30:52 -0500

Update openldap scripts to run behind reverse proxy


---
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index 38e7fdc6b..11afcddab 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -51,3 +51,5 @@ haproxy_port_args:
 - "{{ buildbot_ip_addr }}:443:443"
 - "{{ nexus_ip_addr }}:80:80"
 - "{{ nexus_ip_addr }}:443:443"
+- "{{ phpldapadmin_ip_addr }}:80:80"
+- "{{ phpldapadmin_ip_addr }}:443:443"
